File: C_Programming Assignment/Assignment_24/programAQ2.c
///////////////////////////////////////////////////////////
//
//  File name : Assigment24Question2.c
//  Descreption : Accept string from user and count number of Small characters.
//  Author : Kartik Ganesh Jare
//  Date : 1/6/25
//  
///////////////////////////////////////////////////////////

/*
Input : Marvellous

Output : 9
*/

#include<stdio.h>

int CountSmall(char *str)
{
    int iCount = 0;

    if(str == NULL)
    {
        return -1;
    }

    while(*str != '\0')
    {
        if(*str>= 'a' && *str <= 'z')
        {
            iCount++;
        }
        str++;
    }

    return iCount;
}

int main()
{
    char arr[20] = {'\0'};
    int iRet = 0;

    printf("Enter string :");
    scanf("%[^'\n']s",arr);

    iRet = CountSmall(arr);

    printf("%d",iRet);

    return 0;
}

File: Java_Assignment/Assignment_55/programAQ2.java
///////////////////////////////////////////////////////////
//
//  File name : Assignement55Question2.java
//  Description : Write Java program which accept array of characters from user and
//                replace each capital character with its corresponding small character.
//  Author : Kartik Ganesh Jare
//  Date : 13/11/25
//  
///////////////////////////////////////////////////////////

/*

Input : b N e B R b A i G i
Output : 4

*/

import java.util.Scanner;

class MyArray
{
    public int ArrayCountVowels(char arr[]) 
    {
        int i = 0,iCount = 0;
        char ch = '\0';

        for(i = 0; i < arr.length;i++)
        { 
            ch = arr[i];

            if(ch == 'A' || ch == 'E' || ch == 'I' || ch == 'O' || ch == 'U' &&
                ch == 'a' || ch == 'e' || ch == 'i' || ch == 'o' || ch == 'u'
            )
            {
                iCount ++;
            }
        }

        return iCount;
    }
}

public class programAQ2
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        int iSize = 0,i = 0;
        
        System.out.print("Enter size of array: ");
        iSize = sobj.nextInt();

        char arr[] = new char[iSize];

        System.out.println("Enter elements of array:");
        for(i = 0; i < iSize; i++)
        {
            arr[i] = sobj.next().charAt(0);
        }

        MyArray aobj = new MyArray();
        
        int iRet = aobj.ArrayCountVowels(arr);

        System.out.println(iRet);

        sobj.close();
    }   
}